packages/block-editor/src/hooks/auto-inserting-blocks.js

```diff
@@ -151,7 +151,11 @@ function AutoInsertingBlocksControl( props ) {
 
 	return (
 		<InspectorControls>
-			<PanelBody title={ __( 'Plugins' ) } initialOpen={ true }>
+			<PanelBody
+				className="auto-inserting-blocks-panel"
```

Comment:
In the past, it was recommended to follow BEM for class names, example:

https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/hooks/position.scss

It's documented in:

https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/coding-guidelines.md#naming

I don't see it's followed closely these days, so feel free to decide what works for you best here.

Reply:
Ah, good point! Addressed in 1eb64845f8. Does that seem like an okay name? Or should it include the word `panel` (or something else)?

Reply:
There is no pattern I could distill from other panels, so we can go with what you coded 😄